.github/ISSUE_TEMPLATE/2-bot-commands.yml

```
@@ -6,7 +6,7 @@ body:
       value: |
         Please enter the bot command in the title of the issue and do not modify it further. The conda-forge bots will then trigger the relevant action.
         
-        In case nothing happens, please feel free to reach out to us via our [Matrix](https://matrix.to/#/#conda-forge:matrix.org) community chat room.
+        In case nothing happens, [please feel free to reach out]( https://conda-forge.org/community/getting-in-touch/ ) community chat.
```

Noticed we should drop that last bit

```suggestion
        In case nothing happens, [please feel free to reach out]( https://conda-forge.org/community/getting-in-touch/ ).
```

Handling this with PR: https://github.com/conda-forge/.github/pull/519

---

Ack. Sorry to merge too fast!

---

No it is ok. I had approved too

Only noticed when looking at the rendered version after